etc/ReplayOfflineConfiguration.py

```diff
@@ -1642,6 +1648,43 @@
 ignoreStream(tier0Config, "streamL1Rates")
 ignoreStream(tier0Config, "streamDQMRates")
 
+# Ingore following streams only for this replay
```

Comment:
@francescobrivio this is just to speed up the replay, do I understand it correctly?

Reply:
correct! all streams (except PPS ones) are ignored, so it should be faster!